docs/docs/10.9-perf.ja-JP.md

```diff
@@ -20,6 +20,8 @@ Reactは普通、従来の枠を超えてとても速いです。しかし、ア
 ### `Perf.start()` と `Perf.stop()`
 測定の開始/終了です。その間のReactの操作は以下のような分析のために記録されます。あまり時間を使わない操作は無視されます。
 
+停止した後、あなたは、測定結果を得るために`Perf.getLastMeasurements（）`（後述）が必要になります。
```

Comment:
Can you fix the code piece a bit here to remove the extraneous whitespace (around the parens) and put a space before the opening backtick?